spec/dummy/db/schema.rb

```diff
@@ -2,11 +2,11 @@
 # of editing this file, please use the migrations feature of Active Record to
 # incrementally modify your database, and then regenerate this schema definition.
 #
-# Note that this schema.rb definition is the authoritative source for your
-# database schema. If you need to create the application database on another
-# system, you should be using db:schema:load, not running all the migrations
-# from scratch. The latter is a flawed and unsustainable approach (the more migrations
-# you'll amass, the slower it'll run and the greater likelihood for issues).
+# This file is the source Rails uses to define your schema when running `rails
```

Comment:
Don't check in the changes in this file -- If I run tests with an older version of Rails, it will revert this file back to the old version.  If you want to ignore it, untrack changes locally (google `git --assume-unchanged` for how to do this).